microcosm_pubsub/tests/test_codecs.py

```diff
@@ -103,4 +103,7 @@ def test_decode_missing_field():
     message = dumps({
         "mediaType": DerivedSchema.MEDIA_TYPE,
     })
-    assert_that(calling(codec.decode).with_args(message), raises(ValidationError))
+    assert_that(
+        calling(codec.decode).with_args(message),
+        raises(ValidationError, r".*DerivedSchema"),
```

Comment:
slight update to unit-test to confirm part of exception payload now includes schema name